sdk/app_cpu1/common/drv/eddy_current_sensor.h

```diff
@@ -8,6 +8,8 @@
 
 #include "xparameters.h"
 
+#define AXI_CLK_FREQ_MHZ 200
```

Comment:
Can you rename this to be `FPGA_CLK_FREQ_MHZ` instead of "AXI"

Reply:
In general, these kinds of defines should go in a "global" header file, and not in each module... but for now, I am fine to keep it like this... Eventually, we need to figure out how to pull this value straight from the `xparameters.h` or similar type of auto-gen file. This can be a later PR to update all the IP C code drivers